### ISSUE TITLE
Published Jakarta EE 8 BOM uses non-existing properties

### ISSUE THREAD

**Issue by mjrasche** [eclipse-ee4j/jakartaee-api#97]
The published [jakarta.jakartaee-bom:8.0.0](https://mvnrepository.com/artifact/jakarta.platform/jakarta.jakartaee-bom/8.0.0) artifact is missing the property `jakarta-persistence-api.version`. This was fixed in https://github.com/eclipse-ee4j/jakartaee-api/pull/49. Is a maintenance release planned to fix this issue for Jakarta EE 8 like mentioned in [this](https://github.com/eclipse-ee4j/jakartaee-api/issues/42#issuecomment-574750229) comment?

**Comment by kwsutter** [comment]
Yes, you are correct that the missing property was fixed via PR #49.  So, it was resolved for our 9.0.0 release, but there have not been any requests to release an updated BOM for 8.0.x.  The [comment](https://github.com/eclipse-ee4j/jakartaee-api/issues/42#issuecomment-574750229) you referenced just indicates that it would be possible to create an 8.0.x release, but there was no commitment to do so.

If you feel this is a requirement, either re-purpose this Issue or create a new Issue with the needed justification.  Thanks!

**Comment by zalavaari** [comment]
What shoud we do if we like to have a 8.0.x release with it's current content? :)

**Comment by kwsutter** [comment]
@zalavaari This was discussed on our Platform Development Call today.  This 8.0.x update looks to be relatively straight forward.  We're just in the final stages of the 9.1 release, so nobody raised their hand to deliver this update right at the moment.  This is on our radar and we should be able to address this request once the 9.1 release settles down.  Thanks!

**Comment by malakeel** [comment]
Thank you for the efforts to fix this issue. 
Any updates about making this available ?

**Comment by keilw** [comment]
The PR #49 was already merged and we published Jakarta EE 10 by now, so why is this open and is it still relevant in 10 or 11?

**Comment by mduerig** [comment]
Still not released :(

**Comment by tmurakam** [comment]
I'm waiting 8.0.1 release too. 
I'd like to apply this bom to all my projects before migrate to Jakarta EE 9+.

**Comment by scottkurz** [comment]
I think a good next step would be for someone to raise this on the weekly Platform meeting call:  https://calendar.google.com/calendar/u/0/embed?src=eclipse-foundation.org_e9ki8t2gc75sh07qdh95c8ofvc@group.calendar.google.com&ctz=Europe/Stockholm     Next time I have a chance I will do so but might not be for a couple weeks.

**Comment by malakeel** [comment]
@scottkurz 
Do we have an update?

**Comment by yeroc** [comment]
Any update on this?  Seems like these BOM artifacts should be published with a build number on the end so that fixes can be pushed without a completely new Jakarta EE version being published?  Eg. 8.0.0-2
